src/dates.ts

```diff
@@ -23,6 +23,15 @@ export enum Months {
   December,
 }
 
+export enum TimeUnit {
+  Second = 1000,
+  Minute = Second * 60,
+  Hour = Minute * 60,
+  Day = Hour * 24,
+  Week = Day * 7,
+  Year = Day * 365,
```

Comment:
Nice, I didn't know you could easily self reference like this inside of enums 👍

Reply:
Ya, that was new for me as well. 😄